pkg/session_recording/terminal.jsx

```diff
@@ -22,8 +22,11 @@
 
     var React = require("react");
     var Term = require("term");
+    let $ = require("jquery");
 
     require("console.css");
+    require("jquery-resizable");
+    require("jquery-resizable/resizable.css");
```

Comment:
I've forgot to add this dependency. I'll add this dependency to package.json now and add to this commit.